### PR TITLE
Expanding auto-formatting

### PR DESCRIPTION
**Pull request by kamilritz**
Expanding auto-formatting to folders `mathlib` and `validation` and files `EKF/vel_pos_fusion.cpp` and `imu_down_sampler.*pp`

Style changes: 
- Increase penalty on breaking line after an assignment
- Add opening brackets for functions on new line.

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -37,7 +37,7 @@ BraceWrapping:
   SplitEmptyRecord: true
   SplitEmptyNamespace: true
 BreakBeforeBinaryOperators: None
-BreakBeforeBraces: Attach
+BreakBeforeBraces: Custom # Modified
```
Comment (jkflying) [inline .clang-format]: This seems really inconsistent now. Is there a reason you made it custom?

Comment (kamilritz) [reply .clang-format]: Yes. To allow breaking before braces after functions. These breaks are widely used in the current code bode.

File: EKF/RingBuffer.h [hunk 2]
```diff
@@ -86,12 +88,14 @@ class RingBuffer {
 		return true;
 	}
 
-	void unallocate() {
+	void unallocate()
+	{
```
Comment (dagar) [inline EKF/RingBuffer.h]: This one actually stands out as being different than PX4.

Comment (dagar) [reply EKF/RingBuffer.h]: From the google style guide.

 - "The open curly brace is always on the end of the last line of the function declaration, not the start of the next line."
 - "The close curly brace is either on the last line by itself or on the same line as the open curly brac"

Comment (kamilritz) [reply EKF/RingBuffer.h]: Ok, you got me. But let me add an empty line after the function header.

**Pull request merged**